source/extensions/common/wasm/v8/v8.cc

```diff
@@ -11,15 +11,25 @@
 
 #include "absl/container/flat_hash_map.h"
 #include "absl/strings/match.h"
+#include "v8-version.h"
 #include "wasm-api/wasm.hh"
 
+namespace v8 {
+namespace internal {
+extern bool FLAG_wasm_opt;
+} // namespace internal
+} // namespace v8
+
 namespace Envoy {
 namespace Extensions {
 namespace Common {
 namespace Wasm {
 namespace V8 {
 
 wasm::Engine* engine() {
+  // Enable Wasm optimizations.
+  v8::internal::FLAG_wasm_opt = true;
```

Comment:
Oops, it looks that TSan doesn't like this... Should I suppress the warning or drop the optimization flag for the time being (it's unclear if we gain anything measurable from it)?

Reply:
If you aren't sure if it's worthwhile I would just drop it for now.

Reply:
Dropped.